ExposureApp/Podfile.lock

```diff
@@ -31,4 +31,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 69375b492b584c656a9b0d90925d927fa81adb04
 
-COCOAPODS: 1.8.4
+COCOAPODS: 1.9.3
```

Comment:
Maybe Podfile.lock needs to be added to gitignore

Reply:
Agreed !